blackbox/docs/general/builtins/table-functions.rst

```diff
@@ -6,8 +6,9 @@
 Table Functions
 ===============
 
-Table functions are functions that produce a set of rows. They are used like a
-table or subquery in the ``FROM`` clause of a query.
+Table functions are functions that produce a set of rows which are made up of
+either base data types (scalar types) or composite data types (table rows).
```

Comment:
I'd revert this change. I don't understand this "either base data types (scalar types) or composite data types addition. I think this confuses more than it clarifies.